.travis.yml

```diff
@@ -1,20 +1,19 @@
 language: generic
 services:
 - docker
+- mongodb
 before_install:
 - docker build --target test --tag my-test-image .
 install:
 - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
 - unzip terraform_"$TF_VERSION"_linux_amd64.zip
 - sudo mv terraform /usr/local/bin/
 - rm terraform_"$TF_VERSION"_linux_amd64.zip
-- terraform init
 script:
 - docker run --env-file .env.test my-test-image tests/int_unit
-- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e SECRET_KEY=secret_key -e FLASK_SKIP_DOTENV=True -e MONGO_COLLECTION=todos -e WEBHOOK_URL -e client_id -e client_secret -e disable_login=False -e redirect_uri -e MONGODB_CONNECTION_STRING='mongodb://tasnimamiah:nAL7d1Pxfv2l1SXXOhdqhN4YBMijIBYohCMR6RgX70tbTRCjMRIE5Yr7jD0ukpHVwmXJYs9v9HkBZAEjszzxLQ==@tasnimamiah.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@tasnimamiah@' my-test-image tests/e2e
+- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e SECRET_KEY=secret_key -e FLASK_SKIP_DOTENV=True -e MONGO_COLLECTION=todos -e client_id -e client_secret -e disable_login=False -e redirect_uri -e MONGODB_CONNECTION_STRING -e LOG_LEVEL=DEBUG -e LOGGLY_TOKEN my-test-image tests/e2e
```

Comment:
Worth considering whether you want to pass these values into the e2e test - do you want logs from those tests appearing in the same place as your live logs?